code_src/staking/dot/dotArgparserUtil.py

```diff
@@ -0,0 +1,72 @@
+# actions
+# parser arguments
+
+# TODO find a way to pass action as string to add_argument command
```

Comment:
can you elaborate on the purpose of this in a comment

Reply:
yes argparser take an action as argument i want to create a decorator for the repeated arguments i don't want to repeat same codes in Meany places.
Example :
i want to transform this to string so i can used anywhere

- parser.add_argument('-nt', '--number_of_tokens',
> help='The number of DOT you would like to stake to the network.',required=True, type=int ).